pkg/cloud/services/network/vpc_test.go

```diff
@@ -72,16 +73,16 @@ func TestReconcileVPC(t *testing.T) {
 	selection := infrav1.AZSelectionSchemeOrdered
 
 	testCases := []struct {
-		name        string
-		input       *infrav1.VPCSpec
-		expected    *infrav1.VPCSpec
-		expect      func(m *mock_ec2iface.MockEC2APIMockRecorder)
-		expectError bool
+		name    string
+		input   *infrav1.VPCSpec
+		want    *infrav1.VPCSpec
+		expect  func(m *mock_ec2iface.MockEC2APIMockRecorder)
+		wantErr bool
 	}{
 		{
```

Comment:
Can we follow naming conventions for all the test cases as suggested below so that it will be consistent around the file?

Reply:
I have followed this [naming convention](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea) that you shared in new test cases. Changing this across existing test cases can be done separately as it will span across all the files. I have created an issue [here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/3198) for this. Hope this will help :)